concreate/generator.py

```diff
@@ -44,6 +45,35 @@ def prepare_modules(self, descriptor=None):
         """
         if not descriptor:
             descriptor = self.descriptor
+            if 'source' in descriptor:
+                # copy image source
+                ignored_paths = set()
```

Comment:
I would prefer to go with "white listing" parts of the modules - blacklisting is hard to maintain and make concreate a very hard to put to existing repositories (a lot of changes can introduce a noisy commits adding path to blacklist)

Reply:
That's fine, but I'm not sure how that would work for the example here.  Also, it's very problematic to not filter ./target/ since this causes a recursive copy into itself, which could blow up the disk.

Reply:
I've removed filtering of local module folders through an additional commit.